go-controller/pkg/node/healthcheck.go

```diff
@@ -83,3 +87,34 @@ func countLocalEndpoints(ep *kapi.Endpoints, nodeName string) int {
 	}
 	return num
 }
+
+// check for OVS internal ports without any ofport assigned, they are stale ports that must be deleted
+func checkForStaleOVSInterfaces(stopChan chan struct{}) {
+	ticker := time.NewTicker(60 * time.Second)
+
+	for {
+		select {
+		case <-ticker.C:
+			stdout, _, err := util.RunOVSVsctl("--data=bare", "--no-headings", "--columns=name", "find",
+				"interface", "ofport=-1")
+			if err != nil {
+				klog.Errorf("failed to list OVS interfaces with ofport set to -1")
+				continue
+			}
+			if len(stdout) == 0 {
+				continue
+			}
+			values := strings.Split(stdout, "\n\n")
+			for _, val := range values {
+				klog.Errorf("found stale interface %s, so deleting it", val)
```

Comment:
@girishmg can we make this a warning instead? Since we're about to correct the problem.

Reply:
will do